presentation/src/main/java/jjh/movie/app/presentation/category/CategoryFragment.kt

```diff
@@ -0,0 +1,17 @@
+package jjh.movie.app.presentation.category
+
+import android.os.Bundle
+import android.view.View
+import jjh.movie.app.presentation.base.BaseViewBindingFragment
+import jjh.movie.app.presentation.databinding.FragmentCategoryBinding
+
+class CategoryFragment :
+    BaseViewBindingFragment<FragmentCategoryBinding>(FragmentCategoryBinding::inflate) {
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        initView()
+    }
+
```

Comment:
```suggestion
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        initView()
    }
```
자식에서 override로 새롭게 재정의를 할 때 super로 안전하게 상위 호출을 해두는 것이 좋을 것 같아요!

Reply:
여기서 나도 조금 더 첨언해보자면 왜 super.onViewCreated()를 해주는게 좋을지에 대해 생각해보면 좋을 것 같아.
실제로 super.onViewCreated()는 호출해주지 않아도 동작이 달라지진 않는데 그 이유도 무엇인지 고민해서 코멘트 남겨줘

어떤 것은 super.함수를 사용하고 어떤건 안하는지 이해하고 사용하는게 도움이 될 것 같네~

Reply:
나도 궁금ㅎ

Reply:
상위 클래스인 BaseViewBindingFragment와 Fragment 클래스의 `onViewCreated()` 내부에 코드 블럭이 비어 있어서 호출하지 않아도 동작이 달라지지 않는것 같아요
`super.onViewCreated()`를 해줘야하는 이유는 만약 지금처럼 super.onViewCreated()을 호출해주지 않으면BaseViewBindingFragment의 onViewCreated()에 작성된 코드(현재는 비어있지만)가 불리지 않아서인 것 같습니다..!

Reply:
💯

Reply:
멋집니다.